src/plugins/data_table/dataTables.colReorder.js

```diff
@@ -920,6 +933,165 @@
       this.s.dt.oInstance.oApi._fnSaveState(this.s.dt);
     },
 
+    /**
+     * Initialize sticky table header
+     *  @method  _fnSetupStickyTableHeader
+     *  @param   Int offsetTop number of pixels from top to stick header
+     *  @returns void
+     *  @private
+     */
+     "_fnSetupStickyTableHeader" : function(offsetTop) {
+       if (!offsetTop) { return } else if (offsetTop === true) {
+         offsetTop = 0;
+       }
+
+       // User must trigger 'tableLoaded' event after DataTable has been appended to DOM
+       // We need this to calculate offsets correctly
+       $(window).off('tableLoaded', start.bind(this));
+       $(window).on('tableLoaded', start.bind(this));
+
+       function start() {
+         var th, minWidth, minX;
+         var $window           = $(window);
+         var navbarHeight      = offsetTop;
+         var tableWrapper      = $(this.s.dt.nTableWrapper.children[0]);
+         var table             = $(this.s.dt.nTable);
+         var thead             = $(this.s.dt.nTHead);
+         var theadTop          = tableWrapper.offset().top;
+         var tbody             = $(this.s.dt.nTBody);
+         var tbodyTr           = tbody.children('tr');
+         var theadTh           = thead.children('tr').children('th');
+         var tableBorderOffset = tableWrapper.css('border-left') ? parseInt(tableWrapper.css('border-left').match('.*px')[0].replace('px', '')) : 0;
```

Comment:
this looks brittle (and hard to read).. is that going to work in all cases? can we at least make it it's own method and I'd feel more comfortable handling the case where match returns `null` (because in general it can..)